src/components/SelectPokemons/SelectPokemons.tsx

```diff
@@ -10,9 +10,15 @@ type Props = {
   selectedPokemonSize: number;
   disabledPokemons: string[];
   setIsSelected: () => void;
+  setPlayer: (name: string, url: string) => void;
 };
 
-export const SelectPokemons = ({ disabledPokemons, selectedPokemonSize, setIsSelected }: Props): JSX.Element => {
+export const SelectPokemons = ({
+  disabledPokemons,
+  selectedPokemonSize,
+  setIsSelected,
+  setPlayer,
```

Comment:
I havn't noticed this change in previous PR.

Why have you moved the logic into `SelectPokemons` again? We moved it outside and make the component simpler, better, configurable.

Think on how to refactor the component.